### PR TITLE
[SMALLFIX] Add OBS constants and property keys.

### PR DESCRIPTION
**Pull request by uronce-cc**

### CODE REVIEW

File: core/common/src/main/java/alluxio/PropertyKey.java [hunk 1]
```diff
@@ -553,6 +553,13 @@ public String toString() {
       .setDescription("The access key of GCS bucket.").build();
   public static final PropertyKey GCS_SECRET_KEY = new Builder(Name.GCS_SECRET_KEY)
       .setDescription("The secret key of GCS bucket.").build();
+  public static final PropertyKey OBS_ACCESS_KEY = new Builder(Name.OBS_ACCESS_KEY)
+      .setDescription("The access key of OBS bucket.").build();
+  public static final PropertyKey OBS_ENDPOINT = new Builder(Name.OBS_ENDPOINT)
+      .setDefaultValue("obs.myhwclouds.com")
```
Comment (apc999) [inline core/common/src/main/java/alluxio/PropertyKey.java]: do we want a fake default value here?
we can skip setting default value (=null)

File: core/common/src/main/java/alluxio/Constants.java [hunk 2]
```diff
@@ -62,6 +62,8 @@
   public static final String SCHEME_FT = "alluxio-ft";
   public static final String HEADER_FT = SCHEME_FT + "://";
 
+  public static final String HEADER_OBS = "obs://";
```
Comment (apc999) [inline core/common/src/main/java/alluxio/Constants.java]: "s3n" has been moved to the extension repo. do you want to have "obs" here?

File: core/common/src/main/java/alluxio/PropertyKey.java (hunk 1 above)
Comment (uronce-cc) [reply core/common/src/main/java/alluxio/PropertyKey.java]: it's a meaningful default value according to huawei cloud's doc.

File: core/common/src/main/java/alluxio/Constants.java (hunk 2 above)
Comment (uronce-cc) [reply core/common/src/main/java/alluxio/Constants.java]: removed

File: shell/src/main/java/alluxio/cli/GetConfKey.java [hunk 3]
```diff
@@ -32,6 +32,8 @@
       .put("AWS_SECRETKEY", "aws.secretKey")
       .put("FS_GCS_ACCESSKEYID", "fs.gcs.accessKeyId")
       .put("FS_GCS_SECRETACCESSKEY", "fs.gcs.secretAccessKey")
+      .put("FS_OBS_ACCESSKEY", "fs.obs.accessKey")
+      .put("FS_OBS_SECRETKEY", "fs.obs.secretKey")
```
Comment (apc999) [inline shell/src/main/java/alluxio/cli/GetConfKey.java]: add `OBS_ENDPOINT`?

File: core/common/src/main/java/alluxio/PropertyKey.java (hunk 1 above)
Comment (apc999) [reply core/common/src/main/java/alluxio/PropertyKey.java]: sg

File: core/common/src/main/java/alluxio/PropertyKey.java [hunk 4]
```diff
@@ -751,6 +751,13 @@ public String toString() {
       .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
       .setScope(Scope.SERVER)
       .build();
+  public static final PropertyKey OBS_ACCESS_KEY = new Builder(Name.OBS_ACCESS_KEY)
```
Comment (apc999) [inline core/common/src/main/java/alluxio/PropertyKey.java]: @madanadit  how do we deal with these property keys?
Actually I could not find where we define property keys like `UNDERFS_GLUSTERFS_VOLUMES`.
neither in here or in the ext repo.
Can you clarify ?

File: shell/src/main/java/alluxio/cli/GetConfKey.java (hunk 3 above)
Comment (uronce-cc) [reply shell/src/main/java/alluxio/cli/GetConfKey.java]: `FS_OBS_ENDPOINT` can be correctly transformed into `fs.obs.endpoint`, so it does not need to be put into this map.

File: core/common/src/main/java/alluxio/PropertyKey.java (hunk 4 above)
Comment (madanadit) [reply core/common/src/main/java/alluxio/PropertyKey.java]: @apc999 this is broken. i will create a PR w/ the fix

**Pull request merged**